Extension/src/LanguageServer/extension.ts

```diff
@@ -382,8 +382,7 @@ function onDidChangeVisibleTextEditors(editors: readonly vscode.TextEditor[]): v
     editors.forEach(async (editor) => {
         if (util.isCpp(editor.document)) {
             const client: Client = clients.getClientFor(editor.document.uri);
-            await client.ready;
-            await processDelayedDidOpen(editor.document);
+            await client.queue(()=> processDelayedDidOpen(editor.document));
```

Comment:
There's no space before => but I see a space before the => in other spots.

Reply:
O_o 

I wonder why the formatter fixes this sometimes and not others.  I will see if I can find out why and make it conform

Reply:
The existing code has space before => (I think that is preferred).